media-gfx/blender/blender-2.79b.recipe

```diff
@@ -210,9 +208,9 @@ BUILD()
 		-DWITH_CODEC_AVI=ON \
 		-DWITH_CODEC_FFMPEG=ON \
 		-DWITH_CODEC_SNDFILE=ON \
-		-DWITH_CYCLES=ON \
-		-DWITH_CYCLES_OPENSUBDIV=ON \
-		-DWITH_CYCLES_OSL=ON \
+		-DWITH_CYCLES=OFF \
```

Comment:
What does this mean? why is this disabled?

Reply:
CYCLES build works with older OIIO 1.8.x, but needs patch for OIIO 2.x (API change).

NOTE: tbb was an issue with Blender 2.80, not Blender 2.79b.

Reply:
OK

Reply:
Cycles is a render engine, and while Blender on Haiku is not production ready, i would not disable it, as it is a nice way to compare performance.

Reply:
Otherwise if this workarounds required, you should move this to the BLENDER_BUGOUS section, because the release comes with cycles enabled:
https://git.blender.org/gitweb/gitweb.cgi/blender.git/blob/HEAD:/CMakeLists.txt#l407

I categorized the build settings in the recipe because i wanted feature parity with other platforms, so i marked each and every setting where we cannot follow the official release settings.
Now if you go and change things in place you make my work kaputt, thus i would like to ask: will you maintain the blender recipe in the future? If so, then do what you want, but if not, then please read and change the recipe according the existing settings-categories and do not just change things in-place.